README.md

```diff
@@ -139,7 +145,25 @@ Now, you can navigate to the [Test Interface](https://app.genez.io/test-interfac
 
 Once you are happy with your project, you can deploy it in a production environment by running: `genezio deploy`.
 
-# Examples deployed with `genezio`
+# Commands Summary
+
+The genezio CLI tool supports the commands shown in the following tablke:
+
+| Command | Description |
+| ------- | ----------- |
+| genezio init | Initializes a new project and prepares your project for deploying with genezio. |
```

Comment:
I would add documentation link to each command

Reply:
Done. I added a "Learn more" link to every command.